src/pages/seller/STransactionDetailPage.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    
+    <h2>Transaction Details</h2>
+    <br>
+     <b-list-group deck>
+      <b-list-group-item  v-for="list in listing"
+        v-bind:key="list.id" 
+        class="d-flex justify-content-between align-items-center">
+        <h1 class="mb-1">Buyer Name: {{ list.buyerId }}
+        <br>
+        Quantity: {{ list.quantity }}
+        <br> 
+        <small>Created at: {{ list.createdAt.toDate().toLocaleDateString() }}</small>
+        </h1>
+        
+        <b-button variant="outline-info" class="ml-auto" v-on:click="contact()">Contact Buyer</b-button>
+      
+      </b-list-group-item>
+
+  </b-list-group>
+
+    <br><br>
+    
+    <b-button v-on:click="back()">Back </b-button>
+  </div>
+</template>
+
+<script>
+import { getTransactionsBySeller } from "@/services/transaction.service";
+import { store } from "@/stores";
+import { router } from "@/routes";
+
+export default {
+  name: "STransactionDetailPage",
+  computed: {
+    listing() {
+      return store.getters.getList;
+    },
+  },
+  async created() {
+    const hi = await getTransactionsBySeller(); //change to getTransactionsBySeller
```

Comment:
Do you mean getTransactions over here? getTransactionsBySeller has a required input of userId